frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js

```diff
@@ -113,40 +113,40 @@ export default function ProofOfCompletedService_Web({ data }) {
           listed in section 3 by mail or commerical courier service to:
         </p>
         <TextInput
-          dataKey="proofofcompletedservice_person.fullname"
+          dataKey="person.proofOfCompletedServiceMailFullname"
           label="Name of Addressee"
         />
         <TextInput
-          dataKey="proofofcompletedservice_person.addressStreet"
+          dataKey="person.proofOfCompletedServiceMailAddressStreet"
```

Comment:
same here i believe you can use person.addressStreet etc. Unless I'm wrong and there's a reason this should be a different data item

Reply:
As I indicated above, I believe this person to whom the court document was mailed is not the same person as the plaintiff/petitioner. So, I don't believe we want it to key off the same person? Maybe I'm thinking about this wrong and it won't auto-populate the same name. Is the ```person``` data key just to align formatting for all names, or is it tied to one name (like plaintiff) across all forms?

Reply:
> I believe this person to whom the court document was mailed is not the same person as the plaintiff/petitioner

Yes, this is correct ^ we should not be using `person.addressStreet`

Reply:
Corrected to use `case` data key.